files/bootstrap-fileinput/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "bootstrap-fileinput",
+  "repo": "kartik-v/bootstrap-fileinput",
+  "files": {
+    "include": [ "css/fileinput.min.css", "js/fileinput.min.js" ]
```

Comment:
Add `./` in front of each path to make sure the bot will create the folders. This way the structure will match your repo